---
layout: page
title: Résumé
#featured_image: /assets/images/pages/about.jpg
---

# Skills

#### Computational Proficiencies
- R & Bioconductor
- Linux & Bash
- Docker
- Git & Github
- AWS/Azure/DigitalOcean
- Nextflow
- MySQL
- Parallel/Cluster Computing
- Python & Perl
- Adobe Photoshop & Illustrator

#### Data Analysis Skills
- WGS (variation)
- RNA-seq
- ATAC-seq
- MNase-seq
- DNase-seq
- ChIP-seq
- De novo assembly/annotation
- WGBS
- Targeted sequencing
- Shotgun metagenomics
- 16s amplicon sequencing
- Hi-C, 4C
- Repli-seq
- CRISPR(GeCKO/CRISPRa/CRISPRi) screens
- Pacbio/nanopore sequencing
- RNA/Methyl/ChIP microarray analysis

#### Laboratory Skills
- Molecular biology
- Assay development
- Illumina sequencing
- Nanopore sequencing
- FISH, immunofluorescence
- Fluorescence Microscopy
- Sequence Capture
- Microarray hybridization
- Cloning, plasmid design

# Education
**Doctor of Philosophy (PhD)** in Cell and Molecular Biology, Florida State University, 2014
**Bachelor of Science (BS)** in Biology, Florida State University, 2010

# Experience
### Consultant, silico LLC

I am sole owner and employee of silico, a Massachusetts-based consulting firm. I offer services in **genomics and molecular biology**, including **experimental design, data generation, and data analysis**.

- Developed an algorithm to identify putative novel microproteins in the human and mouse genome.
Combined in-house-generated **shotgun proteomics** data and publicly-available **GWAS/RNA-seq** data to
score and prioritize candidate proteins for chemical/recombinant synthesis and cell-based functional
screening.
*Client: Life Biosciences, LLC*

- Generated **second and third-generation sequencing genome sequencing** and **RNA-seq** data on a novel fungus genome displaying clinically-desirable enzymatic activities. Completely **assembled** (telomere to telomere) and **annotated** the genome to identify putative protein-coding genes. Combined the annotated genome with **shotgun proteomics** to identify candidate proteins responsible for the desirable properties for subsequent biochemical screening in order to enable the development of a recombinantly-manufactured therapeutic.
*Client: Liberty Biosecurity, LLC*

- Developed a novel, rapid, equipment-free, highly-efficient DNA extraction technique deployable in space. Applied this DNA extraction technique on low-biomass environmental samples and performed **shotgun metagenomics** for microbial species abundance measurements and source-tracking contaminants of sterilization-resistant bacteria isolated from NASA spacecraft assembly facilities. Manuscript under review.
*Client: Liberty Biosecurity, LLC.*

- Applied illumina sequencing, and **Hi-C** scaffolding, to **assemble** (telomere to telomere) the muscadine grape genome. Manuscript in preparation.
*Client: Florida A&M University.*

- Presented progress and findings directly to Scientific Advisory Boards, Executive Boards, and Chief
Executive/Scientific Officers.

## Research Fellow, Harvard Medical School
I am currently a Research Fellow at Harvard Medical School, where I am studying the **molecular mechanisms of aging**. I generate, analyze, and integrate large-scale genomic datasets to understand the molecular mechanisms of aging with the ultimate goal of exploiting this information to develop **anti-aging interventions**.

#### Awards
- Awarded the Harvard Translational Research in Aging Training Program (T32) Postdoctoral Award.

#### Publications

- Analyzed RNA-seq, ATAC-seq, ChIP-seq, Hi-C, and other genomic data to characterize the molecular alterations in the genome, epigenome, 3D chromatin-conformation, and transcriptome in mice during aging and in a DNA-damage-based mouse model of aging. [Hayano, M., Yang, J. H., Bonkowski, M. S., Amorim, J. A., Ross, J. M., Coppotelli, G., ... & Vera, D. L. (2019). DNA Break-Induced Epigenetic Drift as a Cause of Mammalian Aging. (preprint)](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3466338)

- Analyzed RNA-seq, ATAC-seq, ChIP-seq, Hi-C, and other genomic data to characterize the molecular alterations in the genome, epigenome, 3D chromatin-conformation, and transcriptome in mouse embryonic fibroblasts. [Yang, J. H., Griffin, P. T., Vera, D. L., Apostolides, J. K., Hayano, M., Meer, M. V., ... & Bhakta, M. (2019). Erosion of the Epigenetic Landscape and Loss of Cellular Identity as a Cause of Aging in Mammals. (preprint)](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3461780)

- Analyzed RNA-seq data to discover the rejuvenation of transcriptomes in retinal ganglion cells of old mice
subjected to AAV-mediated transient overexpression of the Yamanaka factors. [Lu, Y., Krishnan, A., Brommer, B., Tian, X., Meer, M., Vera, D. L., ... & Yang, J. H. (2019). Reversal of ageing-and injury-induced vision loss by Tet-dependent epigenetic reprogramming. BioRxiv, 710210. (preprint)](https://www.biorxiv.org/content/10.1101/710210v1)

- Developed a low-cost and scalable approach to measure epigenetic (D NA methylation ) age in human and
mouse to enable high-throughput chemical and genetic screens.

## Director, Center for Genomics, Florida State University
From 2015 to 2018, I started and directed the **Center for Genomics** at Florida State University. I provided genomic data generation and analysis services for researchers, ran wo rkshops on data generation and analysis, and consulted on all aspects of genomic experiments, including experimental design. I developed new pipelines and algorithms to analyze and integrate diverse types of data and generated much of the data for these projects.

#### Awards
- Awarded (co-PI) an NSF Plant Genome Research Program Grant ([IOS #1444532](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1444532)). Nuclease Profiling as an Integrative Resource for Maize Epigenomics (2015 - 2019).

#### Publications

- Generated and analyzed whole-genome MNase-seq data and identified novel intergenic loci contributing to important agronomic traits in maize. [Rodgers-Melnick, E., Vera, D. L., Bass, H. W., & Buckler, E. S. (2016). Open chromatin reveals the functional maize genome. Proceedings of the National Academy of Sciences, 113(22), E3177-E3184.](https://www.pnas.org/content/113/22/E3177.short)

- Generated and analyzed RNA-seq data to characterize the transcriptional response of iPS-derived hepatocyte-like cells to Dengue virus for the modeling of Dengue-induced hepatotoxicity. [Lang, J., Vera, D., Cheng, Y., & Tang, H. (2016). Modeling dengue virus-hepatic cell interactions using human pluripotent stem cell-derived hepatocyte-like cells. Stem cell reports, 7(3), 341-354.](https://www.sciencedirect.com/science/article/pii/S2213671116301369)

- Analyzed replication timing sequencing (Repli-seq) data in mouse xenografts of pediatric leukemia patient samples and discovered DNA replication timing patterns in clonally-expanded cell populations of patient tumors maintained in serial mouse xenografts. [Sasaki, T., Rivera-Mulia, J. C., Vera, D., Zimmerman, J., Das, S., Padget, M., ... & Weng, A. P. (2017). Stability of patient-specific features of altered DNA replication timing in xenografts of primary human acute lymphoblastic leukemia. Experimental hematology, 51, 71-82.](https://www.sciencedirect.com/science/article/pii/S0301472X17301364)

- Performed WGS in a zebrafish mutant to discover an intergenic regulatory element controlling the expression of a photoreceptor patterning gene that when deleted, causes blindness in a zebrafish model of a human retinopathy. [Sotolongo-Lopez, M., Alvarez-Delfin, K., Saade, C. J., Vera, D. L., & Fadool, J. M. (2016). Genetic dissection of dual roles for the transcription factor six7 in photoreceptor development and patterning in zebrafish. PLoS genetics, 12(4), e1005968.](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005968)

- Developed a data analysis pipeline for Repli-seq data employed by the 4D Nucleome Consortium. [Marchal, C., Sasaki, T., Vera, D., Wilson, K., Sima, J., Rivera-Mulia, J. C., ... & Gilbert, D. M. (2018). Genome-wide analysis of replication timing by next-generation sequencing with E/L Repli-seq. Nature protocols, 13(5), 819.](https://www.nature.com/articles/nprot.2017.148)

- Developed an allele-specific genomic data analysis pipeline and applied it to characterize the allele-specific Repli-seq patterns in genetically hybrid mouse embryonic stem cells and during in vitro differentiation. [Rivera-Mulia, J. C., Dimond, A., Vera, D., Trevilla-Garcia, C., Sasaki, T., Zimmerman, J., ... & Gilbert, D. M. (2018). Allele-specific control of replication timing and genome organization during development. Genome research, 28(6), 800-811.](https://genome.cshlp.org/content/28/6/800.short)

- Analyzed Repli-seq data in Cas9-induced deletions of embryonic stem cells to understand the relationship between DNA sequence, topologically-associated domains ( Hi-C ), transcription ( RNA-seq ), and replication timing control. [Sima, J., Chakraborty, A., Dileep, V., Michalski, M., Klein, K. N., Holcomb, N. P., ... & Bartlett, D. A. (2019). Identifying cis elements for spatiotemporal control of mammalian DNA replication. Cell, 176(4), 816-830.](https://www.sciencedirect.com/science/article/pii/S0092867418315617)

- Generated and analyzed RNA-seq data to characterize the transcriptional response of hPSC-derived macrophages to Dengue and Zika virus infection. [Lang, J., Cheng, Y., Rolfe, A., Hammack, C., Vera, D., Kyle, K., ... & Tang, H. (2018). An hPSC-derived tissue-resident macrophage model reveals differential responses of macrophages to ZIKV and DENV infection. Stem cell reports, 11(2), 348-362.](https://www.sciencedirect.com/science/article/pii/S2213671118302698)

- Analyzed Repli-seq and Hi-C data and contributed to the discovery that replication timing and TADs are established during G1 of the cell cycle. [Dileep, V., Ay, F., Sima, J., Vera, D. L., Noble, W. S., & Gilbert, D. M. (2015). Topologically associating domains and their long-range contacts are established during early G1 coincident with the establishment of the replication-timing program. Genome research, 25(8), 1104-1113.](http://genome.cshlp.org/content/25/8/1104.short)

- Generated and analyzed RNA-seq data to characterize a mutant that disrupts the circadian rhythm of mice. [D’Alessandro, M., Beesley, S., Kim, J. K., Jones, Z., Chen, R., Wi, J., ... & Lee, C. (2017). Stability of wake-sleep cycles requires robust degradation of the PERIOD protein. Current Biology, 27(22), 3454-3467.](https://www.sciencedirect.com/science/article/pii/S0960982217313179)

- Generated and analyzed MNase-seq data to characterize the regulatory elements in a novel crop genome. [Springer, N. M., Anderson, S. N., Andorf, C. M., Ahern, K. R., Bai, F., Barad, O., ... & Buckler, E. S. (2018). The maize W22 genome provides a foundation for functional genomics and transposon biology. Nature genetics, 50(9), 1282.](https://www.nature.com/articles/s41588-018-0158-0)

- Contributed to the development of a novel ChIP-seq peak-finding algorithm. [Girimurugan, S. B., Liu, Y., Lung, P. Y., Vera, D. L., Dennis, J. H., Bass, H. W., & Zhang, J. (2018). iSeg: an efficient algorithm for segmentation of genomic and epigenomic data. BMC bioinformatics, 19(1), 131.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2140-3)

- Contributed to the development of a novel data analysis approach for RNA-seq data. [Wesolowski, S., Vera, D., & Wu, W. (2017). SRSF shape analysis for sequencing data reveal new differentiating patterns. Computational biology and chemistry, 70, 56-64.](https://www.sciencedirect.com/science/article/pii/S1476927117300038)

#### Unpublished software

- Created a customized UCSC genome browser for maize genomes. [http://genomaize.org/](http://genomaize.org/)
- Created a suite of tools for rapid, parallelized data analysis in R. [https://github.com/dvera/travis](https://github.com/dvera/travis)
- Created dockerized UCSC genome browser for data visualization. [https://github.com/dvera/cruize/](https://github.com/dvera/cruize/)

#### Workshops

- Organized and ran 2-day RNA-seq workshop as part of NSF IOS 1354775.
- Organized and ran a 1-day ChIP-seq data analysis workshop.
- Organized and ran **two** 2-week Plant Chromatin Structure workshop (bench to bioinformatics).
- Organized and ran a 1-day workshop on using the UCSC Genome Browser Workshop.
- Organized and helped run an NEB RNA-seq Library Prep Workshop.
- Organized and ran a 2-hour "Tools of the Trade" Bioinformatics Workshop.

## Graduate Research Assistant, Florida State University

During my graduate training (2010-2014) I developed an approach to profile **nucleosome positioning, nucleosome
sensitivity, and transcription factor binding** simultaneously with MNase. I developed user-friendly software
in R to analyze and integrate different genomic data sets. I also participated in multiple collaborations
with other research groups at Florida State University on the generation and analysis of genomic data
sets.

#### Publications

- Developed a novel MNase-seq assay to discover a special class of nucleosomes which vary in their
sensitivity to nuclease digestion and associate with gene regulatory signatures. [Vera, D. L., Madzima, T. F., Labonne, J. D., Alam, M. P., Hoffman, G. G., Girimurugan, S. B., ... & Bass, H. W. (2014). Differential nuclease sensitivity profiling of chromatin reveals biochemical footprints coupled to gene expression and functional DNA elements in maize. The Plant cell, 26(10), 3883-3893.](http://www.plantcell.org/content/26/10/3883.short)

- Through the analysis of Repli-seq data, contributed to the discovery that topologically-associated ( Hi-C ) domains are the functional units regulating mammalian DNA replication timing. [Pope, B. D., Ryba, T., Dileep, V., Yue, F., Wu, W., Denas, O., ... & Thurman, R. E. (2014). Topologically associating domains are stable units of replication-timing regulation. Nature, 515(7527), 402.](https://www.nature.com/articles/nature13986)

- Through the analysis of MNase-chip data, contributed to the identification of widespread and transient repositioning of nucleosomes mediating the inflammatory response in human cells. [Sexton, B. S., Avey, D., Druliner, B. R., Fincher, J. A., Vera, D. L., Grau, D. J., ... & Zhang, J. (2014). The spring-loaded genome: nucleosome redistributions are widespread, transient, and DNA-directed. Genome research, 24(2), 251-259.](http://genome.cshlp.org/content/24/2/251.short)

- Through the analysis of MNase-chip data, contributed to the identification of widespread and transient repositioning of nucleosomes mediating the inflammatory response in human cells. [Sexton, B. S., Druliner, B. R., Vera, D. L., Avey, D., Zhu, F., & Dennis, J. H. (2016). Hierarchical regulation of the genome: global changes in nucleosome organization potentiate genome response. Oncotarget, 7(6), 6460.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4872727/)

- Through the analysis of MNase-chip data, contributed to the identification of widespread and transient repositioning of nucleosomes mediating the inflammatory response in drosophila cells. [Ren, Y., Vera, D. L., Hughes, K. A., & Dennis, J. H. (2015). Stimulation of the Drosophila immune system alters genome-wide nucleosome occupancy. Genomics data, 3, 146-147.](https://www.sciencedirect.com/science/article/pii/S2213596015000021)

- Contributed to the development of an SVM-based algorithm to predict nucleosome positioning using DNA
sequence alone. [Fincher, J. A., Vera, D. L., Hughes, D. D., McGinnis, K. M., Dennis, J. H., & Bass, H. W. (2013). Genome-wide prediction of nucleosome occupancy in maize reveals plant chromatin structural features at genes and other elements at multiple scales. Plant physiology, 162(2), 1127-1141.](http://www.plantphysiol.org/content/162/2/1127.short)

- Performed telomere length measurements to Identify genomic loci associated with telomere length regulation in maize. [Brown, A. N., Lauter, N., Vera, D. L., McLaughlin-Large, K. A., Steele, T. M., Fredette, N. C., & Bass, H. W. (2011). QTL mapping and candidate gene analysis of telomere length control factors in maize (Zea mays L.). G3: Genes, Genomes, Genetics, 1(6), 437-450.](https://www.g3journal.org/content/1/6/437)

- Analyzed genome-scale MNase-chip data during lung adenocarcinoma progression and identified large-scale transient chromatin dynamics associated with tumor progression. [Druliner, B., Fincher, J., Sexton, B., Vera, D., Roche, M., Lyle, S., & Dennis, J. (2013). Chromatin patterns associated with lung adenocarcinoma progression. Cell cycle, 12(10), 1536-1543.](https://www.tandfonline.com/doi/abs/10.4161/cc.24664)

- Analyzed genome-scale MNase-chip data during lung adenocarcinoma progression and identified large-scale transient chromatin dynamics associated with tumor progression. [Druliner, B. R., Vera, D., Johnson, R., Ruan, X., Apone, L. M., Dimalanta, E. T., ... & Dennis, J. H. (2016). Comprehensive nucleosome mapping of the human genome in cancer progression. Oncotarget, 7(12), 13429.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924652/)
