
I am a computational biologist with 10 years of extensive dry and wet research experience. Throughout my career, I have analyzed and integrated genomic and other biological information to understand processes driving genome regulation, disease, and phenotypic variation. I apply intensive exploratory data analysis, data visualization, and statistical approaches to address large-scale problems using genomic, epigenomic, transcriptomic, and proteomic data. I rely heavily on my inherent resourcefulness to tackle new and challenging problems, while simultaneously highly-collaborative and capable of communicating extremely well with individuals from diverse areas of expertise.

# Intrested in working with me?


#### Computational Proficiencies
- R & Bioconductor
- Linux & Bash
- Docker
- Git & Github
- AWS/Azure/DigitalOcean
- Nextflow
- MySQL
- Parallel/Cluster Computing
- Python & Perl
- Adobe Photoshop & Illustrator

#### Data Analysis Skills
- WGS (variation)
- RNA-seq
- ATAC-seq
- MNase-seq
- DNase-seq
- ChIP-seq
- De novo assembly/annotation
- WGBS
- Shotgun metagenomics
- 16s amplicon sequencing
- Hi-C, 4C
- Repli-seq
- CRISPR(GeCKO/CRISPRa/CRISPRi) screens
- Pacbio/nanopore sequencing
- RNA/Methyl/ChIP microarray analysis

#### Laboratory Skills
- Molecular biology
- Assay development
- Illumina sequencing
- Nanopore sequencing
- FISH, immunofluorescence
- Fluorescence Microscopy
- Sequence Capture
- Microarray hybridization
- Cloning, plasmid design
