<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    
        <!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Résumé | Daniel Vera</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Résumé" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="ConsultantComputational Biologist" />
<meta property="og:description" content="ConsultantComputational Biologist" />
<link rel="canonical" href="http://localhost:4000/cv" />
<meta property="og:url" content="http://localhost:4000/cv" />
<meta property="og:site_name" content="Daniel Vera" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Résumé" />
<meta name="twitter:site" content="@" />
<script type="application/ld+json">
{"description":"ConsultantComputational Biologist","@type":"WebPage","url":"http://localhost:4000/cv","headline":"Résumé","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->

    

    <!-- Site Favicon -->
    <link rel="shortcut icon" href="http://localhost:4000/assets/images/favicon.ico" type="image/png" />

    <!-- Font Embed Code -->
	<link href="https://fonts.googleapis.com/css?family=Crimson+Text:400,400i,600,600i|Karla:400,400i,700,700i" rel="stylesheet">

    <!-- CSS Styles -->
    <link href="/assets/css/style.css" rel="stylesheet">
</head>



<body class="layout-page">
    <div id="page" class="site">
        <header id="masthead" class="site-header">
    <div class="site-header-wrap">
        <div class="site-header-inside">

            <div class="site-branding">
                
                <p class="profile">
                    <a href="/">
                        <img src="/assets/images/authorimage.jpg" alt="'s Picture"
                            class="avatar" />
                    </a>
                </p>
                <div class="site-identity">
                    
                    <h1 class="site-title">
                        <a href="/">Daniel Vera</a>
                    </h1>
                    
                    
                    <p class="site-description">Consultant<br>Computational Biologist</p>
                    
                </div><!-- .site-identity -->
                
                <button id="menu-toggle" class="menu-toggle"><span class="screen-reader-text">Main Menu</span><span
                        class="icon-menu" aria-hidden="true"></span></button>
            </div><!-- .site-branding -->

            <nav id="main-navigation" class="site-navigation" aria-label="Main Navigation">
                <div class="site-nav-wrap">
                    <div class="site-nav-inside">
                    <ul class="menu">
                        
                        
                        
                        <li class="menu-item "><a href="/">bio</a></li>
                        
                        
                        
                        <li class="menu-item "><a href="/blog">blog</a></li>
                        
                        
                        
                        <li class="menu-item "><a href="/hire">hire</a></li>
                        
                        
                        
                        <li class="menu-item current-menu-item"><a href="/cv">résumé</a></li>
                        
                        
                        
                        <li class="menu-item "><a href="/contact">Contact Me</a></li>
                        
                    </ul>
                    <p class="social-links">
    
    <a href="mailto:daniel@silico.org" target="_blank">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"/></svg>
<!--
Font Awesome Free 5.5.0 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
-->
    </a>
    
    
    <a href="https://twitter.com/_Daniel_Vera" target="_blank">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg>
<!--
Font Awesome Free 5.5.0 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
-->
    </a>
    
    
    
    <a href="https://github.com/dvera" target="_blank">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
<!--
Font Awesome Free 5.5.0 by @fontawesome - https://fontawesome.com
License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
-->
    </a>
    
    
    
    
    
    
    
    
    
</p>

                    </div><!-- .site-nav-inside -->
                </div><!-- .site-nav-wrap -->
            </nav><!-- .site-navigation -->

        </div><!-- .site-header-inside -->
    </div><!-- .site-header-wrap -->
</header><!-- .site-header -->
        <div id="content" class="site-content fadeInDown delay_075s">
            <div class="inner-wide">
                <main id="main" class="site-main">
    <article class="post-full inner">
        <header class="post-header">
            <h1 class="post-title">Résumé</h1>
        </header><!-- .post-header -->
        
        <div class="post-content inner">
            <h1 id="skills">Skills</h1>

<h4 id="bench-and-bioinformatic-skills">Bench and Bioinformatic Skills</h4>
<ul>
  <li>DNA-seq</li>
  <li>RNA-seq</li>
  <li>ATAC-seq</li>
  <li>MNase-seq</li>
  <li>DNase-seq</li>
  <li>ChIP-seq</li>
  <li>Hi-C, 4C</li>
  <li>WGBS</li>
  <li>De novo assembly/annotation</li>
  <li>Targeted sequencing</li>
  <li>Shotgun metagenomics</li>
  <li>16s amplicon sequencing</li>
  <li>Repli-seq</li>
  <li>CRISPR(GeCKO/CRISPRa/CRISPRi) screens</li>
  <li>Pacbio/nanopore sequencing</li>
  <li>Molecular biology</li>
  <li>Assay development</li>
  <li>Illumina sequencing</li>
  <li>Nanopore sequencing</li>
  <li>FISH, immunofluorescence</li>
  <li>Fluorescence Microscopy</li>
  <li>Sequence Capture</li>
  <li>Microarray hybridization</li>
  <li>Cloning, plasmid design</li>
</ul>

<h4 id="computational-proficiencies">Computational Proficiencies</h4>
<ul>
  <li>R &amp; Bioconductor</li>
  <li>Linux &amp; Bash</li>
  <li>Docker</li>
  <li>Git &amp; Github</li>
  <li>AWS/Azure/DigitalOcean</li>
  <li>Nextflow</li>
  <li>MySQL</li>
  <li>Parallel/Cluster Computing</li>
  <li>Python &amp; Perl</li>
  <li>Adobe Photoshop &amp; Illustrator</li>
</ul>

<h1 id="education">Education</h1>
<p><strong>Doctor of Philosophy (PhD)</strong> in Cell and Molecular Biology, Florida State University, 2014</p>

<p><strong>Bachelor of Science (BS)</strong> in Biology, Florida State University, 2010</p>

<h1 id="experience">Experience</h1>
<h3 id="consultant-silico-llc">Consultant, silico LLC</h3>

<p>I am sole owner and employee of silico, a Massachusetts-based consulting firm. I offer services in <strong>genomics and molecular biology</strong>, including <strong>experimental design, data generation, and data analysis</strong>.</p>

<ul>
  <li>
    <p>Developed an algorithm to identify putative novel microproteins in the human and mouse genome.
Combined in-house-generated <strong>shotgun proteomics</strong> data and publicly-available <strong>GWAS/RNA-seq</strong> data to
score and prioritize candidate proteins for chemical/recombinant synthesis and cell-based functional
screening.
<em>Client: Life Biosciences, LLC</em></p>
  </li>
  <li>
    <p>Generated <strong>second and third-generation sequencing genome sequencing</strong> and <strong>RNA-seq</strong> data on a novel fungus genome displaying clinically-desirable enzymatic activities. Completely <strong>assembled</strong> (telomere to telomere) and <strong>annotated</strong> the genome to identify putative protein-coding genes. Combined the annotated genome with <strong>shotgun proteomics</strong> to identify candidate proteins responsible for the desirable properties for subsequent biochemical screening in order to enable the development of a recombinantly-manufactured therapeutic.
<em>Client: Confidential</em></p>
  </li>
  <li>
    <p>Developed a novel, rapid, equipment-free, highly-efficient DNA extraction technique deployable in space. Applied this DNA extraction technique on low-biomass environmental samples and performed <strong>shotgun metagenomics</strong> for microbial species abundance measurements and source-tracking contaminants of sterilization-resistant bacteria isolated from NASA spacecraft assembly facilities. Manuscript under review.
<em>Client: Liberty Biosecurity, LLC.</em></p>
  </li>
  <li>
    <p>Applied illumina sequencing, and <strong>Hi-C</strong> scaffolding, to <strong>assemble</strong> (telomere to telomere) a large plant genome. Manuscript in preparation.
<em>Client: Florida A&amp;M University.</em></p>
  </li>
  <li>
    <p>Presented progress and findings directly to Scientific Advisory Boards, Executive Boards, and Chief
Executive/Scientific Officers.</p>
  </li>
</ul>

<h2 id="research-fellow-harvard-medical-school">Research Fellow, Harvard Medical School</h2>
<p>I am currently a Research Fellow at Harvard Medical School. I generate, analyze, and integrate large-scale genomic datasets to understand the <strong>molecular mechanisms of aging</strong> with the ultimate goal of exploiting this information to develop <strong>anti-aging interventions</strong>.</p>

<h4 id="awards">Awards</h4>
<ul>
  <li>Awarded the Harvard Translational Research in Aging Training Program (T32) Postdoctoral Award.</li>
</ul>

<h4 id="publications">Publications</h4>

<ul>
  <li>
    <p>Analyzed RNA-seq, ATAC-seq, ChIP-seq, Hi-C, and other genomic data to characterize the molecular alterations in the genome, epigenome, 3D chromatin-conformation, and transcriptome in mice during aging and in a DNA-damage-based mouse model of aging. <a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3466338">Hayano, M., Yang, J. H., Bonkowski, M. S., Amorim, J. A., Ross, J. M., Coppotelli, G., … &amp; Vera, D. L. (2019). DNA Break-Induced Epigenetic Drift as a Cause of Mammalian Aging. (preprint)</a></p>
  </li>
  <li>
    <p>Analyzed RNA-seq, ATAC-seq, ChIP-seq, Hi-C, and other genomic data to characterize the molecular alterations in the genome, epigenome, 3D chromatin-conformation, and transcriptome in mouse embryonic fibroblasts. <a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3461780">Yang, J. H., Griffin, P. T., Vera, D. L., Apostolides, J. K., Hayano, M., Meer, M. V., … &amp; Bhakta, M. (2019). Erosion of the Epigenetic Landscape and Loss of Cellular Identity as a Cause of Aging in Mammals. (preprint)</a></p>
  </li>
  <li>
    <p>Analyzed RNA-seq data to discover the rejuvenation of transcriptomes in retinal ganglion cells of old mice
subjected to AAV-mediated transient overexpression of the Yamanaka factors. <a href="https://www.biorxiv.org/content/10.1101/710210v1">Lu, Y., Krishnan, A., Brommer, B., Tian, X., Meer, M., Vera, D. L., … &amp; Yang, J. H. (2019). Reversal of ageing-and injury-induced vision loss by Tet-dependent epigenetic reprogramming. BioRxiv, 710210. (preprint)</a></p>
  </li>
  <li>
    <p>Developed a low-cost and scalable approach to measure epigenetic (D NA methylation ) age in human and
mouse to enable high-throughput chemical and genetic screens.</p>
  </li>
</ul>

<h2 id="director-center-for-genomics-florida-state-university">Director, Center for Genomics, Florida State University</h2>
<p>From 2015 to 2018, I started and directed the <strong>Center for Genomics</strong> at Florida State University. I provided genomic data generation and analysis services for researchers, ran workshops on data generation and analysis, and consulted on all aspects of genomic experiments, including experimental design. I developed new pipelines and algorithms to analyze and integrate diverse types of data and generated much of the data for these projects.</p>

<h4 id="awards-1">Awards</h4>
<ul>
  <li>Awarded (co-PI) an NSF Plant Genome Research Program Grant (<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1444532">IOS #1444532</a>). Nuclease Profiling as an Integrative Resource for Maize Epigenomics (2015 - 2019).</li>
</ul>

<h4 id="publications-1">Publications</h4>

<ul>
  <li>
    <p>Generated and analyzed whole-genome MNase-seq data and identified novel intergenic loci contributing to important agronomic traits in maize. <a href="https://www.pnas.org/content/113/22/E3177.short">Rodgers-Melnick, E., Vera, D. L., Bass, H. W., &amp; Buckler, E. S. (2016). Open chromatin reveals the functional maize genome. Proceedings of the National Academy of Sciences, 113(22), E3177-E3184.</a></p>
  </li>
  <li>
    <p>Generated and analyzed RNA-seq data to characterize the transcriptional response of iPS-derived hepatocyte-like cells to Dengue virus for the modeling of Dengue-induced hepatotoxicity. <a href="https://www.sciencedirect.com/science/article/pii/S2213671116301369">Lang, J., Vera, D., Cheng, Y., &amp; Tang, H. (2016). Modeling dengue virus-hepatic cell interactions using human pluripotent stem cell-derived hepatocyte-like cells. Stem cell reports, 7(3), 341-354.</a></p>
  </li>
  <li>
    <p>Analyzed replication timing sequencing (Repli-seq) data in mouse xenografts of pediatric leukemia patient samples and discovered DNA replication timing patterns in clonally-expanded cell populations of patient tumors maintained in serial mouse xenografts. <a href="https://www.sciencedirect.com/science/article/pii/S0301472X17301364">Sasaki, T., Rivera-Mulia, J. C., Vera, D., Zimmerman, J., Das, S., Padget, M., … &amp; Weng, A. P. (2017). Stability of patient-specific features of altered DNA replication timing in xenografts of primary human acute lymphoblastic leukemia. Experimental hematology, 51, 71-82.</a></p>
  </li>
  <li>
    <p>Performed WGS in a zebrafish mutant to discover an intergenic regulatory element controlling the expression of a photoreceptor patterning gene that when deleted, causes blindness in a zebrafish model of a human retinopathy. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005968">Sotolongo-Lopez, M., Alvarez-Delfin, K., Saade, C. J., Vera, D. L., &amp; Fadool, J. M. (2016). Genetic dissection of dual roles for the transcription factor six7 in photoreceptor development and patterning in zebrafish. PLoS genetics, 12(4), e1005968.</a></p>
  </li>
  <li>
    <p>Developed a data analysis pipeline for Repli-seq data employed by the 4D Nucleome Consortium. <a href="https://www.nature.com/articles/nprot.2017.148">Marchal, C., Sasaki, T., Vera, D., Wilson, K., Sima, J., Rivera-Mulia, J. C., … &amp; Gilbert, D. M. (2018). Genome-wide analysis of replication timing by next-generation sequencing with E/L Repli-seq. Nature protocols, 13(5), 819.</a></p>
  </li>
  <li>
    <p>Developed an allele-specific genomic data analysis pipeline and applied it to characterize the allele-specific Repli-seq patterns in genetically hybrid mouse embryonic stem cells and during in vitro differentiation. <a href="https://genome.cshlp.org/content/28/6/800.short">Rivera-Mulia, J. C., Dimond, A., Vera, D., Trevilla-Garcia, C., Sasaki, T., Zimmerman, J., … &amp; Gilbert, D. M. (2018). Allele-specific control of replication timing and genome organization during development. Genome research, 28(6), 800-811.</a></p>
  </li>
  <li>
    <p>Analyzed Repli-seq data in Cas9-induced deletions of embryonic stem cells to understand the relationship between DNA sequence, topologically-associated domains ( Hi-C ), transcription ( RNA-seq ), and replication timing control. <a href="https://www.sciencedirect.com/science/article/pii/S0092867418315617">Sima, J., Chakraborty, A., Dileep, V., Michalski, M., Klein, K. N., Holcomb, N. P., … &amp; Bartlett, D. A. (2019). Identifying cis elements for spatiotemporal control of mammalian DNA replication. Cell, 176(4), 816-830.</a></p>
  </li>
  <li>
    <p>Generated and analyzed RNA-seq data to characterize the transcriptional response of hPSC-derived macrophages to Dengue and Zika virus infection. <a href="https://www.sciencedirect.com/science/article/pii/S2213671118302698">Lang, J., Cheng, Y., Rolfe, A., Hammack, C., Vera, D., Kyle, K., … &amp; Tang, H. (2018). An hPSC-derived tissue-resident macrophage model reveals differential responses of macrophages to ZIKV and DENV infection. Stem cell reports, 11(2), 348-362.</a></p>
  </li>
  <li>
    <p>Analyzed Repli-seq and Hi-C data and contributed to the discovery that replication timing and TADs are established during G1 of the cell cycle. <a href="http://genome.cshlp.org/content/25/8/1104.short">Dileep, V., Ay, F., Sima, J., Vera, D. L., Noble, W. S., &amp; Gilbert, D. M. (2015). Topologically associating domains and their long-range contacts are established during early G1 coincident with the establishment of the replication-timing program. Genome research, 25(8), 1104-1113.</a></p>
  </li>
  <li>
    <p>Generated and analyzed RNA-seq data to characterize a mutant that disrupts the circadian rhythm of mice. <a href="https://www.sciencedirect.com/science/article/pii/S0960982217313179">D’Alessandro, M., Beesley, S., Kim, J. K., Jones, Z., Chen, R., Wi, J., … &amp; Lee, C. (2017). Stability of wake-sleep cycles requires robust degradation of the PERIOD protein. Current Biology, 27(22), 3454-3467.</a></p>
  </li>
  <li>
    <p>Generated and analyzed MNase-seq data to characterize the regulatory elements in a novel crop genome. <a href="https://www.nature.com/articles/s41588-018-0158-0">Springer, N. M., Anderson, S. N., Andorf, C. M., Ahern, K. R., Bai, F., Barad, O., … &amp; Buckler, E. S. (2018). The maize W22 genome provides a foundation for functional genomics and transposon biology. Nature genetics, 50(9), 1282.</a></p>
  </li>
  <li>
    <p>Contributed to the development of a novel ChIP-seq peak-finding algorithm. <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2140-3">Girimurugan, S. B., Liu, Y., Lung, P. Y., Vera, D. L., Dennis, J. H., Bass, H. W., &amp; Zhang, J. (2018). iSeg: an efficient algorithm for segmentation of genomic and epigenomic data. BMC bioinformatics, 19(1), 131.</a></p>
  </li>
  <li>
    <p>Contributed to the development of a novel data analysis approach for RNA-seq data. <a href="https://www.sciencedirect.com/science/article/pii/S1476927117300038">Wesolowski, S., Vera, D., &amp; Wu, W. (2017). SRSF shape analysis for sequencing data reveal new differentiating patterns. Computational biology and chemistry, 70, 56-64.</a></p>
  </li>
</ul>

<h4 id="unpublished-software">Unpublished software</h4>

<ul>
  <li>Created a customized UCSC genome browser for maize genomes. <a href="http://genomaize.org/">http://genomaize.org/</a></li>
  <li>Created a suite of tools for rapid, parallelized data analysis in R. <a href="https://github.com/dvera/travis">https://github.com/dvera/travis</a></li>
  <li>Created dockerized UCSC genome browser for data visualization. <a href="https://github.com/dvera/cruize/">https://github.com/dvera/cruize/</a></li>
</ul>

<h4 id="workshops">Workshops</h4>

<ul>
  <li>Organized and ran 2-day RNA-seq workshop as part of NSF IOS 1354775.</li>
  <li>Organized and ran a 1-day ChIP-seq data analysis workshop.</li>
  <li>Organized and ran <strong>two</strong> 2-week Plant Chromatin Structure workshop (bench to bioinformatics).</li>
  <li>Organized and ran a 1-day workshop on using the UCSC Genome Browser Workshop.</li>
  <li>Organized and helped run an NEB RNA-seq Library Prep Workshop.</li>
  <li>Organized and ran a 2-hour “Tools of the Trade” Bioinformatics Workshop.</li>
</ul>

<h2 id="graduate-research-assistant-florida-state-university">Graduate Research Assistant, Florida State University</h2>

<p>During my graduate training (2010-2014) I developed an approach to profile <strong>nucleosome positioning, nucleosome
sensitivity, and transcription factor binding</strong> simultaneously with MNase. I developed user-friendly software
in R to analyze and integrate different genomic data sets. I also participated in multiple collaborations
with other research groups at Florida State University on the generation and analysis of genomic data
sets.</p>

<h4 id="publications-2">Publications</h4>

<ul>
  <li>
    <p>Developed a novel MNase-seq assay to discover a special class of nucleosomes which vary in their
sensitivity to nuclease digestion and associate with gene regulatory signatures. <a href="http://www.plantcell.org/content/26/10/3883.short">Vera, D. L., Madzima, T. F., Labonne, J. D., Alam, M. P., Hoffman, G. G., Girimurugan, S. B., … &amp; Bass, H. W. (2014). Differential nuclease sensitivity profiling of chromatin reveals biochemical footprints coupled to gene expression and functional DNA elements in maize. The Plant cell, 26(10), 3883-3893.</a></p>
  </li>
  <li>
    <p>Through the analysis of Repli-seq data, contributed to the discovery that topologically-associated ( Hi-C ) domains are the functional units regulating mammalian DNA replication timing. <a href="https://www.nature.com/articles/nature13986">Pope, B. D., Ryba, T., Dileep, V., Yue, F., Wu, W., Denas, O., … &amp; Thurman, R. E. (2014). Topologically associating domains are stable units of replication-timing regulation. Nature, 515(7527), 402.</a></p>
  </li>
  <li>
    <p>Through the analysis of MNase-chip data, contributed to the identification of widespread and transient repositioning of nucleosomes mediating the inflammatory response in human cells. <a href="http://genome.cshlp.org/content/24/2/251.short">Sexton, B. S., Avey, D., Druliner, B. R., Fincher, J. A., Vera, D. L., Grau, D. J., … &amp; Zhang, J. (2014). The spring-loaded genome: nucleosome redistributions are widespread, transient, and DNA-directed. Genome research, 24(2), 251-259.</a></p>
  </li>
  <li>
    <p>Through the analysis of MNase-chip data, contributed to the identification of widespread and transient repositioning of nucleosomes mediating the inflammatory response in human cells. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4872727/">Sexton, B. S., Druliner, B. R., Vera, D. L., Avey, D., Zhu, F., &amp; Dennis, J. H. (2016). Hierarchical regulation of the genome: global changes in nucleosome organization potentiate genome response. Oncotarget, 7(6), 6460.</a></p>
  </li>
  <li>
    <p>Through the analysis of MNase-chip data, contributed to the identification of widespread and transient repositioning of nucleosomes mediating the inflammatory response in drosophila cells. <a href="https://www.sciencedirect.com/science/article/pii/S2213596015000021">Ren, Y., Vera, D. L., Hughes, K. A., &amp; Dennis, J. H. (2015). Stimulation of the Drosophila immune system alters genome-wide nucleosome occupancy. Genomics data, 3, 146-147.</a></p>
  </li>
  <li>
    <p>Contributed to the development of an SVM-based algorithm to predict nucleosome positioning using DNA
sequence alone. <a href="http://www.plantphysiol.org/content/162/2/1127.short">Fincher, J. A., Vera, D. L., Hughes, D. D., McGinnis, K. M., Dennis, J. H., &amp; Bass, H. W. (2013). Genome-wide prediction of nucleosome occupancy in maize reveals plant chromatin structural features at genes and other elements at multiple scales. Plant physiology, 162(2), 1127-1141.</a></p>
  </li>
  <li>
    <p>Performed telomere length measurements to Identify genomic loci associated with telomere length regulation in maize. <a href="https://www.g3journal.org/content/1/6/437">Brown, A. N., Lauter, N., Vera, D. L., McLaughlin-Large, K. A., Steele, T. M., Fredette, N. C., &amp; Bass, H. W. (2011). QTL mapping and candidate gene analysis of telomere length control factors in maize (Zea mays L.). G3: Genes, Genomes, Genetics, 1(6), 437-450.</a></p>
  </li>
  <li>
    <p>Analyzed genome-scale MNase-chip data during lung adenocarcinoma progression and identified large-scale transient chromatin dynamics associated with tumor progression. <a href="https://www.tandfonline.com/doi/abs/10.4161/cc.24664">Druliner, B., Fincher, J., Sexton, B., Vera, D., Roche, M., Lyle, S., &amp; Dennis, J. (2013). Chromatin patterns associated with lung adenocarcinoma progression. Cell cycle, 12(10), 1536-1543.</a></p>
  </li>
  <li>
    <p>Analyzed genome-scale MNase-chip data during lung adenocarcinoma progression and identified large-scale transient chromatin dynamics associated with tumor progression. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4924652/">Druliner, B. R., Vera, D., Johnson, R., Ruan, X., Apone, L. M., Dimalanta, E. T., … &amp; Dennis, J. H. (2016). Comprehensive nucleosome mapping of the human genome in cancer progression. Oncotarget, 7(12), 13429.</a></p>
  </li>
</ul>

        </div>
    </article>
    
    <section class="newsletter-box inner">
    <h2 class="newsletter-box-title">Subscribe to Daniel Vera's newsletter</h2>
    <p>Get the latest and greatest from Daniel Vera delivered straight to your inbox every week.</p>
    <!-- Begin MailChimp Signup Form -->
    <div id="mc_embed_signup">
        <form action="https://silico.us5.list-manage.com/subscribe/post?u=8eca20a1e12009844d6caeace&amp;id=6e817b0847" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form"
            class="validate gh-subscribe-form" target="_blank" novalidate>
            <label for="mce-EMAIL" class="screen-reader-text">Email Address</label>
            <input type="email" value="" name="EMAIL" class="required email subscribe-email" id="mce-EMAIL" placeholder="Your email address">
            <input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button">
        </form>
    </div>
    <!--End mc_embed_signup-->
</section><!-- .widget -->

    
</main><!-- .site-main -->


                

                
                <footer id="colophon" class="site-footer">
    <p class="site-info inner">
      Daniel Vera &copy; 2019.

    </p>
    <a id="back-to-top" class="back-to-top" href="#page">
        <span class="icon-arrow-up" aria-hidden="true"></span>
        <span class="screen-reader-text">Back to top</span>
    </a>
</footer><!-- .site-footer -->

            </div><!-- .inner-wide -->
        </div><!-- .site-content -->
    </div><!-- .site -->

    
    <!-- Javascript Assets -->
    <script src="/assets/js/jquery-3.3.1.min.js"></script>
    <script src="/assets/js/plugins.js"></script>
    <script src="/assets/js/custom.js"></script>

</body>

</html>